README.md

```diff
@@ -80,6 +136,16 @@ delta_sharing.load_as_pandas(table_url)
 delta_sharing.load_as_spark(table_url)
 ```
 
+If the table support history sharing, you can query the connector to query table changes.
```

Comment:
support -> supports

you can query the connector to query table changes -> the connector can query table changes.

Reply:
updated.